packages/slate-editor/src/extensions/decorate-selection/components/SelectionHighlight.module.scss

```diff
@@ -1,8 +1,7 @@
 @import "styles/variables";
 
 .selection {
-    border-radius: 4px;
-    box-shadow: 0px 2px 0px 1px $editor-selection-bg, 0px -2px 0px 1px $editor-selection-bg;
```

Comment:
I've also reset the selection decoration styling to match the native browser selection style to avoid styling changes jump when the decoration kicks in.